deployment/cloud_composer.auto.tfvars

```diff
@@ -8,5 +8,9 @@ environment_config = [
   {
     environment_name   = "ml-automation-solutions"
     service_account_id = "ml-auto-solutions"
+  },
+  {
+    environment_name   = "ml-automation-solutions-dev"
+    service_account_id = "ml-auto-solutions-dev"
```

Comment:
Will the dev environment run all of the tests, or do we just expect people to upload one-offs?

Reply:
Sorry that I missed this comment. Good question. I am expecting this env will be the one to test PR in CI/CD. Yes, I think we may still expect people to use their own composer env during the development. WDYT?

Reply:
Do we have a way to stop automated tests from firing in the dev environment so we don't run two entire suites of tests?

People making their own composer environments for testing SGTM. It's hard to coordinate with one environment since the common code is all shared in the bucket.

Reply:
Yes, I was thinking about this. One solution is to put a if/else statement in schedule flag

`schedule="0 6 * * *" if os.environ.get("COMPOSER_ENVIRONMENT") = "ml-auto-solutions" else None`

Reply:
Actually, let me have a try and add into this PR.

Reply:
Added `composer_env` file to hold constants and helper functions. 

I actually tested to update `SCHEDULED_TIME = "0 6 * * *" if composer_env.is_prod_env() else None` to a helper function, but Airflow scheduling does not recognize it. Also if I directly use `"0 6 * * *" if composer_env.is_prod_env() else None` inside of the dag, it throws too many arg error.

* Tested `metric_test.py` passed (plan to set it up in github): https://paste.googleplex.com/4535168667746304
* Uploaded to Airflow, and scheduled is None in dev: https://screenshot.googleplex.com/3JdYht3vcSUvmb6